### DISCUSSION TITLE
Github student pack

### DISCUSSION THREAD

**Discussion by StormAditya** [community/community#110887] [GitHub Education]
### Select Topic Area

Question

### Body

I live in India so most of the schools here don't give school email ids so i had registered using my personal email id and uploaded my ID card on GitHub Developer Pack, it's been more than a day how long will it take and will get approved ?

**Comment by thekarthikr** [comment]
It took me five days to approve.

**Reply by StormAditya** [reply]
thanks alot

**Reply by StormAditya** [reply]
![WhatsApp Image 2024-03-06 at 14 11 40_d035f6c9](https://github.com/community/community/assets/83055431/06ccf050-f37e-4ba8-8d90-7fb40873d6b7)
![Screenshot 2024-03-06 140544](https://github.com/community/community/assets/83055431/87c2ca1d-120d-4d6a-ac26-ceab2b99be86)
(blurred for privacy) can you tell me what's wrong

**Reply by thekarthikr** [reply]
Make sure you applied with a clear picture. If so, try using other academic documents with dates. Change the location on your profile to your school instead of your house. In the billing section, just change the address and add your real name there. I applied for this near my college and with my college email ID.

**Reply by StormAditya** [reply]
![image](https://github.com/community/community/assets/83055431/c758946e-ebfa-4f43-a86f-583cfcc082aa)
well the new one got approved but i wanna confirm that is it now guaranteed i will get the benefits in few days ?

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @StormAditya , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category. I've gone ahead and moved it for you. Good luck!
